### ISSUE TITLE
Debug hover is too narrow and horizontal scrolling useless

### ISSUE THREAD

**Issue by weinand** [microsoft/vscode#97471]
Insiders:

The debug hover is too narrow and horizontal scrolling is useless because it allows only to see a few more characters.

In order to be useful a debug hover must allow horizontal scrolling to the very end of every string.
And it would help if the default size of a debug hover is a 50% wider.

**Comment by isidorn** [comment]
Agree, we could improve this.
However we are not controlling the width, that is done by the Tree.
@joaomoreno is there some way for me as a client to increase the width of each element? I can do it via css and adding a margin, but to check if there is something leaner?

**Comment by weinand** [comment]
I would even tolerate if the width of the tree is set to some hardcoded value (e.g. 1500).

**Comment by joaomoreno** [comment]
> However we are not controlling the width, that is done by the Tree.

What do you mean? You control the width of the container, the tree just fits to 100% of its width.

**Comment by isidorn** [comment]
@joaomoreno yes you are right, but we do not control the width between the end of the element and the start of the vertical scroll bar. Thus what we need is a small width margin at the end each element so it does not overlap the vertical scroll bar. I can do that via css.

**Comment by weinand** [comment]
Please note: the overlap between the content and the vertical scrollbar is not the issue: I want to be able to scroll horizontally to the end of the string!

**Comment by isidorn** [comment]
@weinand ok, sorry I misunderstood you.
@joaomoreno sorry for the ping this issue is on my side.

**Comment by weinand** [comment]
@isidorn works great now!
